### PR TITLE
[13.0][FIX] survey_sale_generation: right partner

### PR DESCRIPTION
**Pull request by chienandalu**
The answer partner should prevail over whatever partner come from the enviroment.

cc @Tecnativa TT40983

please review @stefan-tecnativa @victoralmau @pedrobaeza

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
/ocabot merge patch

File: survey_sale_generation/models/survey_user_input.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class SurveyUserInput(models.Model):
 
     def _prepare_quotation(self):
         return {
-            "partner_id": self.create_uid.partner_id.id,
+            "partner_id": self.partner_id.id or self.create_uid.partner_id.id,
```
Comment (pedrobaeza) [inline survey_sale_generation/models/survey_user_input.py:14]: Oh, wait, `partner_id` field isn't coming from `partner_survey` module?

Comment (chienandalu) [reply survey_sale_generation/models/survey_user_input.py:14]: Nope: https://github.com/odoo/odoo/blob/038354cb17dae43b1144d9f96885445e8bdb6002/addons/survey/models/survey_user.py#L52

**Review by stefan-tecnativa — APPROVED** [review]

**Pull request merged**